corehq/apps/geospatial/views.py

```diff
@@ -446,13 +448,71 @@ class CasesReassignmentView(BaseDomainView):
     def post(self, request, domain, *args, **kwargs):
         request_data = json.loads(request.body)
         case_id_to_owner_id = request_data.get('case_id_to_owner_id', {})
+        include_related_case = request_data.get('include_related_case')
         if len(case_id_to_owner_id) > self.MAX_REASSIGNMENT_REQUEST_CASES:
             return HttpResponseBadRequest(
                 _("Maximum number of cases that can be reassigned is {limit}").format(
-                    self.MAX_REASSIGNMENT_REQUEST_CASES
+                    limit=self.MAX_REASSIGNMENT_REQUEST_CASES
                 )
             )
 
+        if include_related_case:
+            request_cases_id = list(case_id_to_owner_id.keys())
+            parent_to_child_cases_id = self.get_child_cases(domain, request_cases_id)
+            for parent_case_id, child_cases_id in parent_to_child_cases_id.items():
```

Comment:
Nit: rename `child_cases_id` to `child_cases_ids`; this way it's clearer that the second item is a list.

Reply:
Addressed in 0f6157c9be65d299b355e25a906bc8643d4cd278